react/react1/week2/todo-list/src/AddTodo.jsx

```diff
@@ -0,0 +1,31 @@
+import React,{ useState}  from 'react';
+
+import TodoList from "./List";
+import todoList  from './data';
+<todoList todoList = {todoList}/>
+
+export default function AddTodo () { 
```

Comment:
adding todo isn't a component, it's a function of your app, use the naming to describe what your component is

Reply:
I'm not clear about this comment